msteams-platform/bots/build-a-bot.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Let's build a bot
+description: Learn to build a basic chatbot using Teams Toolkit, Bot Framework SDK. Additionally, learn to build an AI bot.
+ms.localizationpriority: high
+ms.date: 09/25/2024
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+# Let's build a bot
+
+Building a bot app involves various phases, from initial concept to final deployment. This article offers a detailed guide to help you get started.
```

Comment:
"This article offers a detailed guide" feels like we're overselling the content here. Maybe walk back the claim for now (until we can provide more content) and say something like:

This article offers a summary of your options to help you get started.

Reply:
@erikadoyle Updated as suggested.